packages/patternfly-4/react-docs/gatsby-config.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const path = require('path');
```

Comment:
This is a build script. Not sure we need to apply linter here? Perhaps a different config for js?

Reply:
Thanks - updated this!